tutorial/practicals/nemo/2_nemo_profiling/README.md

```diff
@@ -49,11 +49,11 @@ this session.)
        $ make allclean
```

Comment:
At line 3, you can use `[Tutorial 1]` to make it a link

Reply:
At line 11 and 12 the links are to the stable version of readthedocs. This is outdated in terms of transformations. I would replace all practicals links to "latest" (as we already do for reference guide links)

Reply:
More good spots.